tests/login_test.py

```diff
@@ -0,0 +1,29 @@
+import time
+import unittest
+
+from selenium import webdriver
+from pages.login_page import LoginPage
+
+
+class LoginTest(unittest.TestCase):
+
+    def setUp(self):
+        baseURL = "https://www.edx.org/"
+        self.driver = webdriver.Firefox()
+        # self.driver = webdriver.Chrome("C:/Users/Aroosha Arif/workspace_python/drivers/chromedriver.exe")
+        self.driver.maximize_window()
+        self.driver.implicitly_wait(3)
+        self.driver.get(baseURL)
+        self.login_page = LoginPage(self.driver)
+
+    def test_valid_login(self):
+        self.login_page.login('newtest5718@gmail.com', 'testselenium123')
```

Comment:
Please add email and password to a new file constants.py and import from there to send here. 
Always better to add passwords as env variables and avoid pushing them to github for security reasons